### DISCUSSION TITLE
Notifications in different language

### DISCUSSION THREAD

**Discussion by dmytrokoren** [jdholtz/auto-southwest-check-in#231] [Ideas]
Would it be possible to do lang/ru/ru.po or something like that.
English of course will be default.

Config file will be lang="ru" if not specified will be English.

msg_successLogin_en "Successfully logged in to {account_monitor.first_name
{account_monitor.last_name}'s account\n"

msg_successLogin_ru "Успешно выполнен вход в учетную запись {account_monitor.first_name
{account_monitor.last_name}\n"

I can do all messages in Russian and Ukrainian if required.

Let me know your thoughts on this?

**Comment by jdholtz** [comment]
Yeah, that would be cool to add. I'll look into setting up language translation for this project.

**Comment by dmytrokoren** [comment]
To add: The language should apply to only apprise notification and not change the debug language.
